pkg/services/authz/client.go

```diff
@@ -181,6 +189,10 @@ func newCloudLegacyClient(authCfg *Cfg, tracer tracing.Tracer) (authlib.AccessCl
 			grpc.WithUnaryInterceptor(clientInterceptor.UnaryClientInterceptor),
 			grpc.WithStreamInterceptor(clientInterceptor.StreamClientInterceptor),
 		),
+		authzlib.WithCacheClientOption(cache.NewLocalCache(cache.Config{
```

Comment:
I don't think we need these changes since we always set up the ttl, and cleanup in the cache wrapper

Reply:
But this is for the client. So what we changed in wrappers won't affect these

Reply:
Ahhh client side, my mistake :pray: